src/Cynthia.Card/src/Cynthia.Card.Server/Locales/en.json

```diff
@@ -3153,27 +3153,27 @@
         },
         "70098": {
             "Name": "Vrihedd Saboteur",
-            "Info": "Play a random Items from your deck. Increase their Damages by 1 each.",
+            "Info": "Deploy: Play a random Items from your deck. Increase their Damages by 1 each.",
             "Flavor": "I can find the bull's-eye from a hundred paces! Perhaps not with my first shot, mind you..."
         },
         "70099": {
             "Name": "Svalblod",
-            "Info": "Damege loyal units in your deck by 2 (doesn't trigger deathwishes), then Strength them by 2.",
+            "Info": "Deploy: Damage loyal units in your deck by 2 (doesn't trigger deathwishes), then Strengthen them by 2.",
             "Flavor": "斯瓦勃洛唯一的戒律：屠戮。”"
         },
         "70100": {
             "Name": "Forest Whisperer",
-            "Info": "Damage all units on the same row of both sides by 2, except for Dryads.",
+            "Info": "Deploy: Damage all units on the same row on both sides by 2, except for Dryads.",
             "Flavor": "布洛克莱昂的树精掌握着一项别致的手艺：随心所欲地变化树形……并赋予它们生命。"
         },
         "70101": {
             "Name": "Immortal Cavalry",
-            "Info": "At the start of the game, lock self. Next time an ally appears Boost it by 3.",
+            "Info": "Single-Use.\nAt the start of the game, lock self. Next time an ally appears Boost it by 3.",
             "Flavor": "骆驼跑得是没有马快……但至少它们在沙漠里能多活两天。"
         },
         "70102": {
             "Name": "Detlaff: Crimson Curse",
-            "Info": "Deploy: Banish 3 Beast or Vampire from your graveyard, then Choose One: Apply 3 rows of Full Moon; or apply 3 rows of Blood Moon.",
+            "Info": "Deploy: Banish 3 Beast or Vampire from your graveyard, then Choose One: Apply  Full Moon on all enemy rows; or apply Blood Moon on all friendly rows.",
```

Comment:
@kajtek14102 The ability description looks incorrect... 
should be "Apply Full Moon on all **friendly** rows; or apply Blood Moon on all **enemy** rows"?